src/tox.ini

```diff
@@ -0,0 +1,22 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
```

Comment:
Is it possible to roll this file's content into `setup.cfg`?  I'd just like to limit the number `.ini` files littered about if I can help it.

Reply:
afaik tox requires a `tox.ini` but it uses the `setup.cfg` and others files to get tools configuration. Like for `pycodestyle`.

Reply:
tox doesn't require a tox.ini (now), setup.cfg can be used with a [tox:tox] section. Leaving this here because it's highly ranked on a search engine. :)